test.js

```diff
@@ -509,6 +535,40 @@ tape('unpack', function (t) {
   t.end()
 })
 
+tape('optioncodes', function (t) {
+  const opts = [
+    [0, 'Reserved 0'],
+    [1, 'LLQ'],
+    [2, 'UL'],
+    [3, 'NSID'],
+    [4, 'Reserved 4'],
+    [5, 'DAU'],
+    [6, 'DHU'],
+    [7, 'N3U'],
+    [8, 'edns-client-subnet', 'CLIENT-SUBNET'],
+    [9, 'EDNS EXPIRE', 'EXPIRE'],
+    [10, 'COOKIE'],
+    [11, 'edns-tcp-keepalive', 'tcp-keepalive'],
+    [12, 'Padding'],
+    [13, 'CHAIN'],
+    [14, 'edns-key-tag', 'key-tag'],
+    [26946, 'DeviceID'],
+    [65535, 'Reserved for future expansion'],
+    [64000, 'Unassigned 64000'],
+    [65002, 'Reserved for Local/Experimental Use 65002'],
+    [-1, 'Invalid -1']
+  ]
+  for (const [code, str, ...aliases] of opts) {
+    const s = optioncodes.toString(code)
+    t.ok(compare(t, s, str), `${code} => ${str}`)
+    t.ok(compare(t, optioncodes.toCode(s), code), `${str} => ${code}`)
+    for (const a of aliases) {
+      t.ok(compare(t, optioncodes.toCode(a), code), `${a} => ${code}`)
+    }
+  }
+  t.end()
+})
+
```

Comment:
While this is exemplary, I think it wil ultimately only makes adding support for new option codes harder while having not much testing value in itself. I'd remove it.

Reply:
Remove the whole test or just the aliases bit?

Reply:
Meant the whole test, but on second thought, it's probably ok to gain some coverage, let's keep it in :wink: